showtimes_gql/mutations/src/projects.rs

```diff
@@ -548,21 +554,24 @@ struct ExternalMediaFetchResult {
     poster_url: Option<String>,
 }
 
-fn unix_to_chrono(unix: i64) -> chrono::DateTime<chrono::Utc> {
-    chrono::DateTime::<chrono::Utc>::from_timestamp(unix, 0).unwrap()
+fn unix_to_timestamp(unix: i64) -> jiff::Timestamp {
+    jiff::Timestamp::from_second(unix).unwrap()
 }
 
-fn fuzzy_yyyy_mm_dd_to_chrono(date: &str) -> Option<chrono::DateTime<chrono::Utc>> {
+fn fuzzy_yyyy_mm_dd_to_timestamp(date: &str) -> Option<jiff::Timestamp> {
     let parts: Vec<&str> = date.split('-').collect();
-    let year: Option<i32> = parts.first().and_then(|y| y.parse().ok());
-    let month: Option<u32> = parts.get(1).and_then(|m| m.parse().ok());
-    let day: Option<u32> = parts.get(2).and_then(|d| d.parse().ok());
+    let year: Option<i16> = parts.first().and_then(|y| y.parse().ok());
+    let month: Option<i8> = parts.get(1).and_then(|m| m.parse().ok());
+    let day: Option<i8> = parts.get(2).and_then(|d| d.parse().ok());
 
     // If all None, return None
     match (year, month, day) {
-        (Some(year), Some(month), Some(day)) => chrono::Utc
-            .with_ymd_and_hms(year, month, day, 0, 0, 0)
-            .single(),
+        (Some(year), Some(month), Some(day)) => {
+            let dt = jiff::civil::datetime(year, month, day, 0, 0, 0, 0);
```

Comment:
I think you can just do `jiff::civil::date(year, month, day)` here. And `civil::Date` has a `to_zoned` method that will use midnight automatically.

Reply:
I would also suggest using the fallible `civil::Date::new`, since I don't see any reason to assume that `year`/`month`/`day` is a valid date.